README.md

```diff
@@ -68,6 +68,19 @@ For non-NixOS systems, install directly with:
 nix-env -iA nixpkgs.tenere
 ```
 
+### 📱 Mobile (nix-on-droid)
+
+Tenere also works on Android via nix-on-droid:
+
+[![Nix-on-droid Demo](assets/tenere-on-droid-thumb.gif)](assets/tenere-on-droid.gif)
```

Comment:
maybe we can remove this line ? the demos will take too much place in my opinion

Reply:
removed the demo gif from readme, linked to it instead.  Added link to tutorial.